recipe/gen_patch_json.py

```diff
@@ -2795,6 +2795,14 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             new_constrains.append(f"cuda-cccl_{subdir} <0.0.0a0")
             record['constrains'] = new_constrains
 
+        if (
+            record_name == "sqlalchemy-cockroachdb" and
+            record["version"] == "2.0.0" and
+            record["build_number"] == 0 and
+            record.get("timestamp", 0) <= 1680784303548
+        ):
+            _replace_pin("sqlalchemy <2.0.0", "sqlalchemy >=2.0.0", record["depends"], record)
```

Comment:
Looks like upstream doesn't pin this: https://github.com/cockroachdb/sqlalchemy-cockroachdb/blob/v2.0.0/setup.py#L39